packages/components-js/public/index.html

```diff
@@ -93,6 +94,9 @@
       <option value="inline-notification-example-action-button">Inline Notification Example Action Button</option>
       <option value="inline-notification-example-events">Inline Notification Example Events</option>
       <option value="modal-example-accessibility">Modal Example Accessibility</option>
+      <option value="multi-select-example">Multi Select Example</option>
+      <option value="multi-select-example-controlled">Multi Select Controlled Example</option>
+      <option value="multi-select-example-dynamic">Multi Select Dynamic Example</option>
```

Comment:
I think we got this naming convention.
```suggestion
      <option value="multi-select-example-controlled">Multi Select Example Controlled</option>
      <option value="multi-select-example-dynamic">Multi Select Example Dynamic</option>
```